content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -287,6 +287,35 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl on macOS
+
+Depending on how you installed `kubectl`, use one of the following methods.
+
+### Uninstall kubectl using the command-line
+
+1. Locate the `kubectl` binary on your system:
+
+```bash
+where kubectl
+```
+
+1. Remove the `kubectl` binary:
+
+```bash
```

Comment:
Indent this entire code block and line 307 by 3 spaces 

(This is needed because otherwise the site generator thinks these are two separate lists instead of one single list)